spec/src/spec.ts

```diff
@@ -1,3 +1,4 @@
+import path = require('path')
```

Comment:
Why is require import necessary here?

Reply:
By mistake. VSCode  auto generated that one.